google/cloud/bigquery/job/base.py

```diff
@@ -171,6 +171,33 @@ def __setattr__(self, name, value):
             )
         super(_JobConfig, self).__setattr__(name, value)
 
+    @property
+    def jobtimeout_ms(self):
+        """ Optional parameter. Job timeout in milliseconds. If this time limit is exceeded, BigQuery might attempt to stop the job.
+
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfiguration.FIELDS.job_timeout_ms
+
+        e.g.
+            job_config = bigquery.QueryJobConfig( jobtimeout = 5000 )
+            
+                or
+
+            job_config.jobtimeout = 5000
```

Comment:
Update the example's syntax to match your new property name.

Reply:
Resolved syntax on 185